### ISSUE TITLE
bfg-ish fails for repo arguments other than '.'

### ISSUE THREAD

**Issue by pberndt** [newren/git-filter-repo#203]
1) bfg-ish apparently assumes at some places to have been invoked from inside repo.

2) Also, where bfg-ish cds into the repo dir, the argument files (e.g. replacement.txt) are searched with relative paths from inside the repo, not relative to bfg-ish invocation cwd, which is unexpected. But even when compensating this by passing ../replacements.txt the command fails because of 1)

```
bfg-ish -fi app -fi '*.{rb}' --replace-text ../replacements.txt myrepo
Processed 1572 blob sizes
Parsed 104 commits
New history written in 0.92 seconds...
HEAD is now at af86408 whitelist 10.250.0.1 for web_console
Completely finished after 0.93 seconds.
Repacking your repo and cleaning out old unneeded objects
Traceback (most recent call last):
  File "/home/user/src/git-filter-repo/contrib/filter-repo-demos/bfg-ish", line 439, in <module>
    bfg.run()
  File "/home/user/src/git-filter-repo/contrib/filter-repo-demos/bfg-ish", line 435, in run
    fr.RepoFilter.cleanup(bfg_args.repo, repack=True, reset=need_another_reset)
  File "/home/user/src/git-filter-repo/contrib/filter-repo-demos/git_filter_repo.py", line 2973, in cleanup
    subproc.call(cmd, cwd=repo)
  File "/usr/lib/python3.7/subprocess.py", line 323, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/usr/lib/python3.7/subprocess.py", line 775, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.7/subprocess.py", line 1522, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: b'myrepo': b'myrepo'

```

Tested with version e9e0308df1e9e792598b8e41d0086d7a785e9cc5

**Comment by Lin-Buo-Ren** [comment]
I've made a patch that supposed to fix the issue, please verify.

**Comment by newren** [comment]
This was fixed as e732141 (Fix relative path compatibility for --replace-text and bfg_args.repo, 2021-04-17), I just forgot to close the associated issue years ago; closing now.
